api/repos.go

```diff
@@ -38,6 +38,7 @@ func apiReposCreate(c *gin.Context) {
 		Comment             string
 		DefaultDistribution string
 		DefaultComponent    string
+		LdapGroup           string
```

Comment:
IIUC, if `LdapGroup` is an empty string, it means "no ldap used in this repo". Maybe it would be best to use `nil` for that purpose?

Reply:
redesigned quite a bit to manage this cleaner